server/src/services/ConversationService.ts

```diff
@@ -0,0 +1,100 @@
+import { MessageParam, ContentBlockParam } from "@anthropic-ai/sdk/resources/messages/messages.mjs";
+
+export class ConversationService {
```

Comment:
nit: I think that because it already belong to the services folder then we don't need to add post-fix Service anymore.

```suggestion
export class Conversation {
```